### DISCUSSION TITLE
Getting RabbitMQ logs in Azure log analytics workspace

### DISCUSSION THREAD

**Discussion by paywagh11** [rabbitmq/cluster-operator#1525] [Q&A]
I have a 3 node RabbitMQ cluster in kubernetes environment with Kubernetes cluster operator and messaging topology operator.

I can see the RMQ pod logs in Azure log analytics workspace.
But I would like to get the RMQ application logs in Azure log analytics workspace as visible in RMQ UI. 

Does anyone have any idea how to configure it or any documentation available?

**Comment by Zerpet** [comment]
We don't have any specific documentation to Azure. Usually, you would ship the logs from rabbit, or any other component in Kubernetes, using a log exporter, like [Fluentd](https://www.fluentd.org/) or fluentbit. If you want to export metrics, similar to RabbitMQ UI, RabbitMQ exposes prometheus metrics; most systems (like Azure) are capable of ingesting this format of metrics, regardless if whether they use Prometheus.
